server/backend/api/app/schemas.py

```diff
@@ -10,35 +10,7 @@
 LOGGER = logging.getLogger(__name__)
 
 
-class FamGroupPost(BaseModel):
```

Comment:
removed the schemas not are not in use anymore, also regroup the schemas a little bit